src/Misc/layoutbin/en-US/strings.json

```diff
@@ -258,6 +258,7 @@
   "EnsureJobFinished": "Waiting for current job finish running.",
   "EnsureTasksExist": "Download all required tasks.",
   "EnterValidValueFor0": "Enter a valid value for {0}.",
+  "EnvironmentVariableExceedsMaximumLength": "Value of environment variable '{0}' exceeds the maximum supported size of {1} characters, task(s) might not be able to use it.",
```

Comment:
Can we keep this text?
Environment variable '{0}' exceeds maximum supported length. Environment variable length: {1}, Maximum supported length: {2}.

Reply:
Used the suggested text.